### PR TITLE
Get composer from official image

### PR DESCRIPTION
**Pull request by yguedidi**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -54,10 +54,9 @@ RUN set -ex \
  && ln -sf /usr/sbin/php-fpm81 /usr/sbin/php-fpm \
  && rm -rf /var/cache/apk/* \
  && ln -sf /dev/stdout /var/log/nginx/access.log \
- && ln -sf /dev/stderr /var/log/nginx/error.log \
- && curl -s https://getcomposer.org/installer | php \
- && mv composer.phar /usr/local/bin/composer \
- && rm -rf /root/.composer/*
+ && ln -sf /dev/stderr /var/log/nginx/error.log
+
+COPY --from=composer:2.5.8 /usr/bin/composer /usr/local/bin/composer
```
Comment (yguedidi) [inline Dockerfile:59]: used a fixed version as a good practice

Comment (yguedidi) [reply Dockerfile:59]: and added after the PHP setup as I think we're more likely to upgrade composer than to change PHP setup

Comment (Kdecherf) [reply Dockerfile:59]: It would be better to use a ARG to specify the version, like for `WALLABAG_VERSION`, it would ease maintenance.

**Review by Kdecherf — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (yguedidi) [reply Dockerfile]: @Kdecherf what about using `composer:2`? so we at least stick to a fixed major version, composer following SemVer it's fine I think

**Review by j0k3r — APPROVED** [review]

**Pull request merged**